nuxt.config.ts

```diff
@@ -9,6 +9,13 @@ export default defineNuxtConfig({
   devServer: {
     port: 8080,
   },
+  nitro: {
+    routeRules: {
+      '/proxy/gate': { proxy: 'https://api.gateio.ws/api/v4/spot/tickers?currency_pair=AE_USDT', cors: true },
```

Comment:
Some of the request needed to be passed through proxy to get over CORS. Some links didn't need this approach